doc/UsersGuide/source/doc-snippets/hsd_data.rst

```diff
@@ -0,0 +1,6 @@
+Data for the HSD cases is already staged on Tier-1 platforms at the ``INPUTROOT_*`` locations listed in `baseline_setup.yaml <https://github.com/ufs-community/ufs-weather-model/blob/develop/tests-dev/baseline_setup.yaml>`_. However, users on any platform can download the data directly from the `HTF data bucket <https://registry.opendata.aws/noaa-ufs-htf-pds/>`_ using ``wget``. 
```

Comment:
data is at the INPUTDATA_ROOT path (e.g., https://github.com/ufs-community/ufs-weather-model/blob/6b0f516557811eac82c17b852efb82a35892b022/tests-dev/baseline_setup.yaml#L11)

think the variable is just wrong here